framework/auth/views.py

```diff
@@ -56,6 +56,12 @@ def reset_password(auth, **kwargs):
 
 
 def forgot_password():
+    """Return forgot password page upon GET request. If POST, attempt to send
+    user password reset or return respective error.
+    """
+    if request.method == 'GET':
```

Comment:
GET requests are still passed through by _forgot_password() -- see the weird error a few comments up